k8s/modules-ingress-mainnet.tf

```diff
@@ -11,7 +11,7 @@ module "ingress-kong_cid-checker-calibrationnet" {
   type_lb_scheme                   = "external"
 }
 
-module "ingress-kong_cid-checker-calibrationnet" {
+module "ingress-kong_cid-checker-another-mainnet" {
   count                            = local.is_mainnet_envs
```

Comment:
I regard you need the `is_kong_auth_header_enabled      = false` for the mainnet env as well